students/MikeShand/lesson04/mailroom_switch.py

```diff
@@ -0,0 +1,93 @@
+#!/usr/bin/env python3
+from collections import defaultdict
+
+# donors = {'Andy': [10.00], 'Bill': [15.00, 25.00], 'Chuck': [20.00, 30.00, 40.00]}
+
+donors = defaultdict(list)
+donor_list = [['Andy', [10.00]], ['Bill', [15.00, 25.00]], ['Chuck', [20.00, 30.00, 40.00]]]
+for name, data in donor_list:
+    donors[name] = data

```

Comment:
this might be coming from a bit of confusion on how to use `defaultdict`?
you can define your initial dict inline without the need for another type of data structure and an extra loop:
```
>>> from collections import defaultdict
>>> donors = defaultdict(list, {"Andy": [10.00]})
>>> donors["natasha"].append(1000)
>>> donors
defaultdict(<class 'list'>, {'Andy': [10.0], 'natasha': [1000]})
>>>

```